### DISCUSSION TITLE
max_length设置与实际不符

### DISCUSSION THREAD

**Discussion by DH-burn** [milvus-io/milvus#28333] [Q&A and General discussion]
我设置了max_length为1023，但实际插入一个长度为376的字符串，却报错我插入了一个长度为1044的字符串超出长度限制。并且我每减少一个字符，报错长度减少3个，这是怎么回事

**Comment by xiaofan-luan** [comment]
note the max_length in on bytes.
For chinese characters it is usually 2-4 bytes
